distribution/src/main/resources/bin/elasticsearch

```diff
@@ -44,6 +44,13 @@
 
 # Check to see if you are trying to run this without building it first. Gradle
```

Comment:
This comment is now in the wrong place, should be after the `jvm_options` function